connector-common-http/src/main/java/com/redhat/cloud/notifications/connector/http/HttpConnectorConfig.java

```diff
@@ -0,0 +1,82 @@
+package com.redhat.cloud.notifications.connector.http;
+
+import com.redhat.cloud.notifications.connector.ConnectorConfig;
+import jakarta.enterprise.context.ApplicationScoped;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+import java.util.List;
+import java.util.Map;
+
+@ApplicationScoped
+public class HttpConnectorConfig extends ConnectorConfig {
+
+    private static final String COMPONENTS = "notifications.connector.http.components";
+    private static final String CONNECT_TIMEOUT_MS = "notifications.connector.http.connect-timeout-ms";
+    private static final String CONNECTIONS_PER_ROUTE = "notifications.connector.http.connections-per-route";
+    private static final String DISABLE_FAULTY_ENDPOINTS = "notifications.connector.http.disable-faulty-endpoints";
+    private static final String FOLLOW_REDIRECTS = "notifications.connector.http.follow-redirects";
+    private static final String MAX_TOTAL_CONNECTIONS = "notifications.connector.http.max-total-connections";
+    private static final String SOCKET_TIMEOUT_MS = "notifications.connector.http.socket-timeout-ms";
+
+    @ConfigProperty(name = COMPONENTS, defaultValue = "http,https")
+    List<String> httpComponents;
+
+    @ConfigProperty(name = CONNECT_TIMEOUT_MS, defaultValue = "2500")
+    int httpConnectTimeout;
+
+    @ConfigProperty(name = CONNECTIONS_PER_ROUTE, defaultValue = "20")
+    int httpConnectionsPerRoute;
+
+    @ConfigProperty(name = DISABLE_FAULTY_ENDPOINTS, defaultValue = "true")
+    boolean disableFaultyEndpoints;
+
+    @ConfigProperty(name = FOLLOW_REDIRECTS, defaultValue = "false")
+    boolean followRedirects;
+
+    @ConfigProperty(name = MAX_TOTAL_CONNECTIONS, defaultValue = "200")
+    int httpMaxTotalConnections;
+
+    @ConfigProperty(name = SOCKET_TIMEOUT_MS, defaultValue = "2500")
+    int httpSocketTimeout;
```

Comment:
180000 only makes sense for connectors that need to talk to `recipients-resolver`.

I'd rather use 2500 as the default and 180000 as the exception rather than the opposite.

Reply:
Ah, undertood! Thank you for the explanation, it makes total sense!